### DISCUSSION TITLE
Modify pyperf to display CPU stats and geomean to one decimal place.

### DISCUSSION THREAD

**Discussion by markshannon** [faster-cpython/ideas#247] [Ideas]
Modern CPUs have a lot of internal counters for things like L1 cache misses, pipeline stalls, etc.
Those counters would be valuable when comparing minor improvements that don't show much speedup.
`perf` can capture those stats, so we should also capture and display them.

It would also be useful to have geomean to one decimal place. There is quite a difference in significance between a 0.6% speedup and a 1.4% speedup.

**Comment by gvanrossum** [comment]
Maybe the next thing to do here is to file an issue with https://github.com/psf/pyperf/issues ?
